### PR TITLE
Added route for text nodes

### PR DESCRIPTION
**Pull request by raffazizzi**
Added a route option for text nodes, in case users want to process text nodes directly. This can be useful to support spanning elements like `<delSpan>` that may affect text nodes, e.g.:

```xml
<delSpan spanTo="#D1"/>
<list>
    <item>item one</item>
    <item>item <anchor xml:id="D1"/> two</item>
</list>
```

Where the text node with value "item" is affected, while its sibling "two" is not. By writing a component that matches text nodes, one can determine which text node is deleted and which isn't and, for example, apply styles.

I chose the keyword `text()` to match text nodes, as it will be familiar to XPath users, but I'm open to other suggestions.

### CODE REVIEW

**Comment by raffazizzi** [comment]
PS I bumped the version number as a minor version rather than a patch since this is a new feature.

File: src/TEINode.js [hunk 1]
```diff
@@ -35,6 +35,24 @@ class TEINode extends React.Component {
     if (!this.props.teiNode) return null
 
     if (this.props.teiNode.nodeType === 3) {
+      if (this.props.availableRoutes.includes('text()')) {
+        const propsClone = {
```
Comment (pfefferniels) [inline src/TEINode.js]: perhaps it is possible to move the following code to its own function – the exact duplication a few lines later is maybe a little bit ugly …?

**Review by pfefferniels — COMMENTED** [review]
Great, `text()` seems fine to me! What do you think, perhaps even an short example (in the `examples/`) for this use-case might be interesting?

**Comment by raffazizzi** [comment]
I've re-organized `TEINode.js` to avoid code repetition -- sorry about that, I sent a PR too quickly! I've changed a few things, all seems good to me, but please test with your project too!

I've added an example for matching text node to render `<delSpan>`. I ended up integrating the CETEIcean XML example so that I could change it, and while I was at it I added a direct link to its CSS for a better looking example.

**Comment by pfefferniels** [comment]
It's looking very good! 👍

**Pull request merged**